File: test/fixtures/javascript-xhr/multiline_post_with_data.js
const data = new URLSearchParams({
  'msg1': 'value1',
  'msg2': 'value2'
});

let xhr = new XMLHttpRequest();
xhr.withCredentials = true;
xhr.open('GET', 'http://localhost:28139/echo/html/');
xhr.setRequestHeader('Origin', 'http://fiddle.jshell.net');
xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');

xhr.onload = function() {
  console.log(xhr.response);
};

xhr.send(data);

File: test/fixtures/java-unirest/multiline_post_with_data.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.get("http://localhost:28139/echo/html/")
            .header("Origin", "http://fiddle.jshell.net")
            .header("Content-Type", "application/x-www-form-urlencoded")
            .field("msg1","value1")
            .field("msg2","value2")
            .asString();
    }
}
